modules/exploits/windows/http/zoho_password_manager_pro_xml_rpc_rce.rb

```diff
@@ -0,0 +1,160 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+
+  Rank = ExcellentRanking
+
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+  include Msf::Exploit::JavaDeserialization
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Apache OFBiz XML-RPC Java Deserialization',
```

Comment:
```suggestion
        'Name' => 'Zoho Password Manager Pro XML-RPC Java Deserialization',
```

Reply:
Fixed in https://github.com/rapid7/metasploit-framework/pull/16852/commits/6d45320c0c885ce8e3725399b8907da78d60c039, sorry I didn't fix this earlier!